pusher/pusher_client.py

```diff
@@ -107,8 +107,23 @@ def trigger_batch(self, batch=[], already_encoded=False):
         """
         if not already_encoded:
             for event in batch:
-                event['data'] = data_to_string(event['data'],
-                self._json_encoder)
+
+                if not isinstance(event['channel'], six.string_types):
+                    raise TypeError("Channel name is expected to be a Python2 str or unicode or a Python 3 str")
+
+                validate_channel(event['channel'])
+
+                event_name = ensure_text(event['name'], "event_name")
+                if len(event['name']) > 200:
+                    raise ValueError("event_name too long")
+
+                event['data'] = data_to_string(event['data'], self._json_encoder)
+
+                if len(event['data']) > 10240:
+                    raise ValueError("Too much data")
```

Comment:
Could you create an issue on this repository to argue that we should remove validation from the SDK (leaving it to the server)? The main points in favour:

1. We can update the limits on the server, and the SDKs will be forwards compatible
2. We might want to have different limits for different users

Reply:
Will do. 👍